src/interfaces/ILockManager.sol

```diff
@@ -33,19 +23,16 @@ struct LockManagerSettings {
 /// @notice Helper contract acting as the vault for locked tokens used to vote on multiple plugins and proposals.
 interface ILockManager {
     /// @notice Returns the current settings of the LockManager.
-    function settings() external view returns (UnlockMode unlockMode, PluginMode pluginMode);
+    function settings() external view returns (PluginMode pluginMode);
 
     /// @notice Returns the address of the voting plugin.
     /// @return The LockToVote plugin address.
     function plugin() external view returns (ILockToGovernBase);
 
     /// @notice Returns the address of the token contract used to determine the voting power.
+    ///     If the native token is being used it returns address(0).
```

Comment:
I think this echoes my thoughts on the Execute condition: having the unitiialised value also be the sentinel for native I think is dangerous. I prefer magic values in this setting to be unambiguous.